cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -674,7 +674,7 @@ public static ClusterOperatorConfig dummyClusterOperatorConfig(KafkaVersion.Look
     }
 
     public static ClusterOperatorConfig dummyClusterOperatorConfig(KafkaVersion.Lookup versions, long operationTimeoutMs) {
-        return dummyClusterOperatorConfig(versions, ClusterOperatorConfig.DEFAULT_OPERATION_TIMEOUT_MS, "");
+        return dummyClusterOperatorConfig(versions, operationTimeoutMs, "");
```

Comment:
while playing around with the tests, I noticed that this method ignores the `operationTimeoutMs` argument and I feel this is a mistake? hence this fix.

Reply:
I think Shubham dealt with it in #8054?